x-pack/plugins/endpoint/public/applications/endpoint/store/policy_list/fake_data.ts

```diff
@@ -50,4 +63,21 @@ export const getFakeDatasourceApiResponse = async (page: number, pageSize: numbe
     page,
     perPage: pageSize,
   };
+
+  savedFakePolicyData = { items: policyData.items };
+  return policyData;
+};
+
+export const getFakeDatasourceDetailsApiResponse = async (id: string) => {
+  await new Promise(resolve => setTimeout(resolve, 500));
+
+  // Emulates the API response - see PR:
+  // https://github.com/elastic/kibana/pull/56567/files#diff-431549a8739efe0c56763f164c32caeeR25
+
+  if (savedFakePolicyData.items) {
```

Comment:
If you extract that function above that generate a single Policy, then here you can use it like: 

```javascript
return {
  ...generateSinglePolicyData('some name');
  ...{ id }
}
```